volume/drivers/proxy.go

```diff
@@ -1,6 +1,6 @@
 // generated code - DO NOT EDIT
```

Comment:
Does something else need to be updated, because this is generated code?

Reply:
No, it just takes the package name from the package it's being run from.